src/app/(social)/my-follower/page.client.tsx

```
@@ -0,0 +1,275 @@
+                      onClick={() => {
+                        follow(index);
+                      }}
+                    />
+                  )
+                )}
+              </div>
+
+              <div
+                id="user-profile-description"
+                className="font-medium text-[12px] text-neutral-300"
+                dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)